library/src/lib/toggle/toggle.component.ts

```diff
@@ -58,37 +77,55 @@ export class ToggleComponent implements OnInit, ControlValueAccessor {
         }
     }
 
+    /** Set focus on the input element. */
     public focus(): void {
         this.inputElement.nativeElement.focus();
     }
 
+    /** Get the innerInputId property of the toggle. */
     get innerInputId(): string {
         return `${this.id}-input`;
     }
 
+    /** Get the isChecked property of the toggle. */
     get isChecked() {
         return this.checked;
     }
 
+    /** Set the isChecked property of the toggle. */
     set isChecked(value) {
         this.checked = value;
         this.onChange(value);
         this.onTouched();
         this.checkedChange.emit(value);
     }
 
+    /**
+     * @param value Sets the value of the *checked* property of the toggle.
+     */
     writeValue(value: any) {
```

Comment:
Can add @hidden here since this is only used by angular forms, but keep the description as it could be useful